Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.4.1
```

Comment:
Check out https://hub.docker.com/r/jekyll/jekyll/ it already includes nodejs and ruby in one image

Reply:
Oh cool, thanks I'll use that!